src/CMakeLists.txt

```diff
@@ -57,11 +52,12 @@ set(libraries_common
     ${CURL_LIBRARIES}
     pthread
     z
+    ${CONAN_LIBS}
```

Comment:
Did the build work with the removed `find_packages` (lines 11-17 above)? Leaving them in the CMakeLists.txt file, without including `${CONAN_LIBS}` allows the project to build with and without Conan, whereas using this variable couples the build to it. If it works with the generic find_packages, I would favour keeping them instead.

Reply:
You're absolutely right. I already restored the ``find_packages`` but not yet pushed this change. Thank you for pointing out

Reply:
:+1: can confirm that this currently breaks non-Conan builds.
Out of curiosity: Why did the corresponding `conan` branch in the `forward-epics-to-kafka` not require `$CONAN_LIBS`?

Reply:
If the `find_packages` commands don't assume any specific location for the packages, it should work with Conan without `$CONAN_LIBS`. That was the case for the Forwarder. If you encounter a problem with this, please let me know.